[2023-06-25T06:16:00.439+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_etl_minio_v15.spark_submit_task scheduled__2023-06-24T00:00:00+00:00 [queued]>
[2023-06-25T06:16:00.457+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_etl_minio_v15.spark_submit_task scheduled__2023-06-24T00:00:00+00:00 [queued]>
[2023-06-25T06:16:00.457+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-25T06:16:00.457+0000] {taskinstance.py:1283} INFO - Starting attempt 2 of 6
[2023-06-25T06:16:00.457+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-25T06:16:00.475+0000] {taskinstance.py:1303} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2023-06-24 00:00:00+00:00
[2023-06-25T06:16:00.480+0000] {standard_task_runner.py:55} INFO - Started process 28679 to run task
[2023-06-25T06:16:00.483+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'spark_etl_minio_v15', 'spark_submit_task', 'scheduled__2023-06-24T00:00:00+00:00', '--job-id', '242', '--raw', '--subdir', 'DAGS_FOLDER/spark_job_minio_v1.py', '--cfg-path', '/tmp/tmpgywv9yc_']
[2023-06-25T06:16:00.484+0000] {standard_task_runner.py:83} INFO - Job 242: Subtask spark_submit_task
[2023-06-25T06:16:00.497+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:250: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-06-25T06:16:00.517+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-06-25T06:16:00.546+0000] {task_command.py:388} INFO - Running <TaskInstance: spark_etl_minio_v15.spark_submit_task scheduled__2023-06-24T00:00:00+00:00 [running]> on host a8d69ab09d2d
[2023-06-25T06:16:00.618+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=spark_etl_minio_v15
AIRFLOW_CTX_TASK_ID=spark_submit_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-24T00:00:00+00:00
[2023-06-25T06:16:00.628+0000] {base.py:73} INFO - Using connection ID 'spark_conn' for task execution.
[2023-06-25T06:16:00.631+0000] {spark_submit.py:341} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.hadoop.fs.s3a.aws.credentials.provider=org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider --conf spark.hadoop.fs.s3a.path.style.access=true --conf spark.hadoop.fs.s3a.access.key=minioadmin --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.endpoint=http://minio:9000 --jars local:///opt/bitnami/spark/jars/hadoop-aws-3.2.0.jar,local:///opt/bitnami/spark/jars/aws-java-sdk-bundle-1.11.375.jar,local:///opt/bitnami/spark/jars/hadoop-common:3.2.0.jar --name arrow-spark /opt/***/dags/spark_etl_spark_minio.py
[2023-06-25T06:16:02.022+0000] {spark_submit.py:492} INFO - WARNING: An illegal reflective access operation has occurred
[2023-06-25T06:16:02.023+0000] {spark_submit.py:492} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/opt/spark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2023-06-25T06:16:02.023+0000] {spark_submit.py:492} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2023-06-25T06:16:02.023+0000] {spark_submit.py:492} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2023-06-25T06:16:02.023+0000] {spark_submit.py:492} INFO - WARNING: All illegal access operations will be denied in a future release
[2023-06-25T06:16:02.349+0000] {spark_submit.py:492} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2023-06-25T06:16:02.350+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:02 WARN DependencyUtils: Local jar /opt/bitnami/spark/jars/hadoop-aws-3.2.0.jar does not exist, skipping.
[2023-06-25T06:16:02.350+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:02 WARN DependencyUtils: Local jar /opt/bitnami/spark/jars/aws-java-sdk-bundle-1.11.375.jar does not exist, skipping.
[2023-06-25T06:16:02.350+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:02 WARN DependencyUtils: Local jar /opt/bitnami/spark/jars/hadoop-common:3.2.0.jar does not exist, skipping.
[2023-06-25T06:16:03.550+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SparkContext: Running Spark version 3.2.0
[2023-06-25T06:16:03.657+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2023-06-25T06:16:03.782+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO ResourceUtils: ==============================================================
[2023-06-25T06:16:03.783+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO ResourceUtils: No custom resources configured for spark.driver.
[2023-06-25T06:16:03.784+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO ResourceUtils: ==============================================================
[2023-06-25T06:16:03.785+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SparkContext: Submitted application: arrow-spark
[2023-06-25T06:16:03.819+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2023-06-25T06:16:03.837+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO ResourceProfile: Limiting resource is cpu
[2023-06-25T06:16:03.838+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2023-06-25T06:16:03.899+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SecurityManager: Changing view acls to: default
[2023-06-25T06:16:03.899+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SecurityManager: Changing modify acls to: default
[2023-06-25T06:16:03.899+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SecurityManager: Changing view acls groups to:
[2023-06-25T06:16:03.900+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SecurityManager: Changing modify acls groups to:
[2023-06-25T06:16:03.904+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:03 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(default); groups with view permissions: Set(); users  with modify permissions: Set(default); groups with modify permissions: Set()
[2023-06-25T06:16:04.335+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO Utils: Successfully started service 'sparkDriver' on port 45343.
[2023-06-25T06:16:04.420+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO SparkEnv: Registering MapOutputTracker
[2023-06-25T06:16:04.485+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO SparkEnv: Registering BlockManagerMaster
[2023-06-25T06:16:04.520+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2023-06-25T06:16:04.524+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2023-06-25T06:16:04.531+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2023-06-25T06:16:04.591+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-01114b42-8a63-46b9-978a-6e359185f2a4
[2023-06-25T06:16:04.620+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2023-06-25T06:16:04.646+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO SparkEnv: Registering OutputCommitCoordinator
[2023-06-25T06:16:04.947+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2023-06-25T06:16:04.948+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2023-06-25T06:16:04.949+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2023-06-25T06:16:04.949+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2023-06-25T06:16:04.950+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2023-06-25T06:16:04.951+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2023-06-25T06:16:04.952+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2023-06-25T06:16:04.953+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2023-06-25T06:16:04.954+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4048. Attempting port 4049.
[2023-06-25T06:16:04.955+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4049. Attempting port 4050.
[2023-06-25T06:16:04.956+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 WARN Utils: Service 'SparkUI' could not bind on port 4050. Attempting port 4051.
[2023-06-25T06:16:04.971+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:04 INFO Utils: Successfully started service 'SparkUI' on port 4051.
[2023-06-25T06:16:05.059+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://a8d69ab09d2d:4051
[2023-06-25T06:16:05.074+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO SparkContext: Added JAR local:///opt/bitnami/spark/jars/hadoop-aws-3.2.0.jar at file:/opt/bitnami/spark/jars/hadoop-aws-3.2.0.jar with timestamp 1687673763540
[2023-06-25T06:16:05.075+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO SparkContext: Added JAR local:///opt/bitnami/spark/jars/aws-java-sdk-bundle-1.11.375.jar at file:/opt/bitnami/spark/jars/aws-java-sdk-bundle-1.11.375.jar with timestamp 1687673763540
[2023-06-25T06:16:05.075+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO SparkContext: Added JAR local:///opt/bitnami/spark/jars/hadoop-common:3.2.0.jar at file:/opt/bitnami/spark/jars/hadoop-common:3.2.0.jar with timestamp 1687673763540
[2023-06-25T06:16:05.369+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2023-06-25T06:16:05.470+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO TransportClientFactory: Successfully created connection to spark/172.19.0.11:7077 after 57 ms (0 ms spent in bootstraps)
[2023-06-25T06:16:05.591+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20230625061605-0008
[2023-06-25T06:16:05.597+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 33829.
[2023-06-25T06:16:05.598+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO NettyBlockTransferService: Server created on a8d69ab09d2d:33829
[2023-06-25T06:16:05.600+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2023-06-25T06:16:05.607+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, a8d69ab09d2d, 33829, None)
[2023-06-25T06:16:05.612+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO BlockManagerMasterEndpoint: Registering block manager a8d69ab09d2d:33829 with 434.4 MiB RAM, BlockManagerId(driver, a8d69ab09d2d, 33829, None)
[2023-06-25T06:16:05.613+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, a8d69ab09d2d, 33829, None)
[2023-06-25T06:16:05.614+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, a8d69ab09d2d, 33829, None)
[2023-06-25T06:16:05.875+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:05 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2023-06-25T06:16:06.244+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:06 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2023-06-25T06:16:06.270+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:06 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2023-06-25T06:16:11.207+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO CodeGenerator: Code generated in 129.318454 ms
[2023-06-25T06:16:11.240+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO SparkContext: Starting job: showString at NativeMethodAccessorImpl.java:0
[2023-06-25T06:16:11.253+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO DAGScheduler: Got job 0 (showString at NativeMethodAccessorImpl.java:0) with 1 output partitions
[2023-06-25T06:16:11.253+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO DAGScheduler: Final stage: ResultStage 0 (showString at NativeMethodAccessorImpl.java:0)
[2023-06-25T06:16:11.253+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO DAGScheduler: Parents of final stage: List()
[2023-06-25T06:16:11.254+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO DAGScheduler: Missing parents: List()
[2023-06-25T06:16:11.257+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO DAGScheduler: Submitting ResultStage 0 (MapPartitionsRDD[6] at showString at NativeMethodAccessorImpl.java:0), which has no missing parents
[2023-06-25T06:16:11.349+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO MemoryStore: Block broadcast_0 stored as values in memory (estimated size 11.7 KiB, free 434.4 MiB)
[2023-06-25T06:16:11.378+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO MemoryStore: Block broadcast_0_piece0 stored as bytes in memory (estimated size 6.2 KiB, free 434.4 MiB)
[2023-06-25T06:16:11.381+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on a8d69ab09d2d:33829 (size: 6.2 KiB, free: 434.4 MiB)
[2023-06-25T06:16:11.384+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO SparkContext: Created broadcast 0 from broadcast at DAGScheduler.scala:1427
[2023-06-25T06:16:11.397+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO DAGScheduler: Submitting 1 missing tasks from ResultStage 0 (MapPartitionsRDD[6] at showString at NativeMethodAccessorImpl.java:0) (first 15 tasks are for partitions Vector(0))
[2023-06-25T06:16:11.398+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:11 INFO TaskSchedulerImpl: Adding task set 0.0 with 1 tasks resource profile 0
[2023-06-25T06:16:26.416+0000] {spark_submit.py:492} INFO - 23/06/25 06:16:26 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2023-06-25T06:16:30.822+0000] {local_task_job.py:276} WARNING - State of this instance has been externally set to None. Terminating instance.
[2023-06-25T06:16:30.831+0000] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 28679. PIDs of all processes in the group: [28680, 28746, 28679]
[2023-06-25T06:16:30.832+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 28679
[2023-06-25T06:16:30.847+0000] {taskinstance.py:1482} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-06-25T06:16:30.865+0000] {spark_submit.py:618} INFO - Sending kill signal to spark-submit
[2023-06-25T06:16:30.892+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1523, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1588, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 411, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 460, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1483, in signal_handler
    raise AirflowTermSignal("Task received SIGTERM signal")
airflow.exceptions.AirflowTermSignal: Task received SIGTERM signal

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1377, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1530, in _execute_task_with_callbacks
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-06-25T06:16:30.939+0000] {taskinstance.py:1321} INFO - Marking task as UP_FOR_RETRY. dag_id=spark_etl_minio_v15, task_id=spark_submit_task, execution_date=20230624T000000, start_date=20230625T061600, end_date=20230625T061630
[2023-06-25T06:16:30.969+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 242 for task spark_submit_task ((psycopg2.errors.ForeignKeyViolation) insert or update on table "task_fail" violates foreign key constraint "task_fail_ti_fkey"
DETAIL:  Key (dag_id, task_id, run_id, map_index)=(spark_etl_minio_v15, spark_submit_task, scheduled__2023-06-24T00:00:00+00:00, -1) is not present in table "task_instance".

[SQL: INSERT INTO task_fail (task_id, dag_id, run_id, map_index, start_date, end_date, duration) VALUES (%(task_id)s, %(dag_id)s, %(run_id)s, %(map_index)s, %(start_date)s, %(end_date)s, %(duration)s) RETURNING task_fail.id]
[parameters: {'task_id': 'spark_submit_task', 'dag_id': 'spark_etl_minio_v15', 'run_id': 'scheduled__2023-06-24T00:00:00+00:00', 'map_index': -1, 'start_date': datetime.datetime(2023, 6, 25, 6, 16, 0, 443809, tzinfo=Timezone('UTC')), 'end_date': datetime.datetime(2023, 6, 25, 6, 16, 30, 935699, tzinfo=Timezone('UTC')), 'duration': 30}]
(Background on this error at: https://sqlalche.me/e/14/gkpj); 28679)
[2023-06-25T06:16:30.987+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=28680, status='terminated', started='06:16:00') (28680) terminated with exit code None
[2023-06-25T06:16:30.989+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=28679, status='terminated', exitcode=1, started='06:16:00') (28679) terminated with exit code 1
[2023-06-25T06:16:30.989+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=28746, status='terminated', started='06:16:02') (28746) terminated with exit code None
